domains/pabitrabanerjee.json

```
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "PB2204",
+           "email": "rockstarpabitra2204@gmail.com",
+           "discord": "1091585182974345366"
+        },
+    
+        "record": {
+            "CNAME": "12.14.243.127"
```

Website is not accessible, please give a preview of your website.
```suggestion
            "A": ["12.14.243.127"]
```

---

URL of my website is : https://pabitrabanerjee.netlify.app/